terraform/hcl/applicable/req_sw_terraform_aws_cloudtrail_encrypt.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_cloudtrail" "vulnerable_example" {
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 5 2 "vulnerable_example" /terraform/hcl/applicable/req_sw_terraform_aws_cloudtrail_encrypt.tf)

## ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableMediumSeverity.png)<br>  medium Infrastructure as Code Vulnerability 🛠️
	
Finding: kms_key_id='' was detected

### 👇 Details

Resource `aws_cloudtrail` should have key `kms_key_id`

Vulnerable example -
```
resource "aws_cloudtrail" "vulnerable_example" {
  # kms_key_id is not set
}
```

Secure example -
```
resource "aws_cloudtrail" "secure_example" {
  kms_key_id = aws_kms_key.example.arn
}
```	

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)